ote/src/cljs/ote/app/controller/pre_notices.cljs

```diff
@@ -36,8 +38,10 @@
 (defmethod routes/on-navigate-event :pre-notices [_]
   (->LoadOrganizationPreNotices))
 
-(defmethod routes/on-navigate-event :authority-pre-notices [_]
-  (->LoadAuthorityPreNotices))
+(defmethod routes/on-navigate-event :authority-pre-notices [{params :params}]
```

Comment:
I think that this is a bit scary. Not because given parameters arent mandatatory but because it call entirely different event. And by event I mean "page". So for me this would be easier to understand if we had
route event for "authority-pre-notices" and "auhtority-pre-notice/:id" (which is not restful naming convention).